src/tests/AttributesManagersTest.cpp

```diff
@@ -597,6 +683,21 @@ TEST_F(AttributesManagersTest, ObjectAttributesManagersCreate) {
   ASSERT_EQ(origNumPrimBased, newNumPrimBased3);
 }  // AttributesManagersTest::ObjectAttributesManagersCreate test
 
+TEST_F(AttributesManagersTest, LightLayoutAttributesManagerTest) {
+  LOG(INFO) << "Starting "
+               "AttributesManagersTest::LightLayoutAttributesManagerTest";
+
+  std::string lightConfigFile = Cr::Utility::Directory::join(
+      DATA_DIR, "test_assets/lights/test_lights.lighting_config.json");
+
+  LOG(INFO) << "Start Test : Create, Edit, Remove Attributes for "
+               "LightLayoutAttributesManager @ "
+            << lightConfigFile;
+  // light attributes manager attributes verifcation
+  testCreateAndRemoveLights(lightLayoutAttributesManager_, lightConfigFile);
```

Comment:
can you point me to the implementation of this so I can take a look? I couldn't find it.

Reply:
Note : this is not creating lights, it is creating light attributes.  The test is [here](https://github.com/facebookresearch/habitat-sim/blob/98f4b98496340d3671bf509d91391cdcd9d3b4f0/src/tests/AttributesManagersTest.cpp#L175)